src/test/scala/com/feedzai/cosytest/junit/IntegrationSpec.java

```diff
@@ -0,0 +1,89 @@
+package com.feedzai.cosytest.junit;
+
+import com.feedzai.cosytest.DockerComposeRule;
+import com.feedzai.cosytest.DockerComposeSetup;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scala.Option;
+import scala.Predef;
+import scala.Some;
+import scala.collection.JavaConverters;
+import scala.collection.Seq;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.is;
+
+
+
+public class IntegrationSpec {
+
+    private static Logger logger = LoggerFactory.getLogger(IntegrationSpec.class);
+
+
+    private static final DockerComposeSetup dockerSetup;
+
+    static {
+        Path workingDirectory = Paths.get("src", "test", "resources", "docker-compose-junit.yml");
+        List<Path> workingDir = new ArrayList<>();
+        workingDir.add(workingDirectory);
+        Map<String, String>environment = new HashMap<>();
+        dockerSetup = new DockerComposeSetup(
+                "junittest",
+                JavaConverters.asScalaIteratorConverter(workingDir.iterator()).asScala().toSeq(),
```

Comment:
Not sure Java people would enjoy this exercise :) Can we have a setup factory with a Java-friendly method, like
```java
dockerSetup = DockerComposeSetupFactory.newInstance(
    "junittest",
    Arrays.asList(...),
    Paths.get(""),
    "ENV1", "env1 value",
    "ENV2", "env2 value",
    ...
)
```